src/themes/svg_path.js

```diff
@@ -0,0 +1 @@
+export const svgPath = './svg/logo.svg';
```

Comment:
Please rename this file to images.js as it more usual name for this file

Reply:
ok

Reply:
I've just do that.